spec/support/features/doc_auth_helper.rb

```diff
@@ -56,11 +56,7 @@ def idv_doc_auth_document_capture_step
   end
 
   def idv_doc_auth_verify_step
-    if IdentityConfig.store.doc_auth_verify_info_controller_enabled
-      idv_verify_info_path
-    else
-      idv_doc_auth_step_path(step: :verify)
-    end
+    idv_verify_info_path
```

Comment:
note to self: I should probably go back and get rid of this whole method and replace it with `idv_verify_info_path` everywhere it is used